src/logging.rs

```diff
@@ -102,7 +112,7 @@ impl Plugin for LogViewerPlugin {
         app.add_event::<LogEvent>();
         app.add_systems(Update, transfer_log_events);
```

Comment:
lets actually remove `transfer_log_events`. we can read out of the receiver right inside `update_log_ui`.